server/lib/src/idm/mod.rs

```diff
@@ -5,6 +5,7 @@
 
 pub mod account;
 pub(crate) mod accountpolicy;
+pub mod application;
```

Comment:
pub(crate), doesn't need to be public to everything.

Reply:
Fixed.